libraries/core-react/stories/components/Popover.stories.tsx

```diff
@@ -51,13 +51,25 @@ export const Default: Story<PopoverProps> = (args) => {
 
   const referenceElement = useRef(null)
 
+  const onKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
+    const { key } = e
+    switch (key) {
+      case 'Escape':
+        handleToggle()
+        break
+      default:
+        break
+    }
+  }
+
```

Comment:
This approach differs from how we have done it in `Menu`, where we do this internally in the component and call the `onClose` callback when key is triggered. Suggest we revert this change and make a seperate PR for having this internally in the component like `Menu`

Reply:
Oops, I just continued with the change on this PR. But renamed the task and PR